tools/autograph-client/client.go

```diff
@@ -166,10 +151,6 @@ examples:
 		log.Printf("signing hash %q", hash)
 		url = url + "/sign/hash"
 		data = hash
-	} else if rsapssHash != "base64(sha1(data))" {
```

Comment:
nit: might be nice to keep this around until we unship rsapss :woman_shrugging: 

ideally we could ship different versions of the client

Reply:
but this isn't needed since `-a` already sets a hash value for signing